api/nvidia/v1/clusterpolicy_types.go

```diff
@@ -841,6 +843,60 @@ type SandboxDevicePluginSpec struct {
 	Env []EnvVar `json:"env,omitempty"`
 }
 
+// DRADriverSpec defines the properties for the NVIDIA DRA Driver deployment
+// TODO: add 'controller' and 'kubeletPlugin' structs to allow for per-component configuration
```

Comment:
One question: Should we expose `controller` and `kubeletPlugin` as concepts to the user? These seem like internal details and including them here couples the operator and the DRA driver implementation more tightly.

Reply:
I don't believe we should at this point in time. But I can see where having the ability to control the `controller` / `kubeletPlugin` configuration independently could be useful. For example, one may need to bump the `cpu` / `mem` resources for the controller to account for larger sized clusters. Open to continue discussion on this and enumerate the list of fields we want to expose in Clusterpolicy.